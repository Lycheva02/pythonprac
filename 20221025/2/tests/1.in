print(*list(slide(range(7, 103, 12), 10)))