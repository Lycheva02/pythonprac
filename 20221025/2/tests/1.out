7 19 31 43 55 67 79 91 19 31 43 55 67 79 91 31 43 55 67 79 91 43 55 67 79 91 55 67 79 91 67 79 91 79 91 91
